Sources/PayPal/Buttons/PayPalButton.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+/// Configuration for PayPal button
+public class PayPalButton: PaymentButton {
```

Comment:
Thought - would it be helpful to differentiate the `PayPalCheckoutButton` from the `PayPalCreditButton`?

Reply:
I wonder if that'd be confusing since merchants would still need to launch the checkout experience on their own and the button is just a static button currently. I do see what you are saying though - I feel like calling the entire SDK PayPal and having a PayPal module / buttons in it is getting kinda confusing. 🤔